src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/sideeffects/GameEventHandler.java

```diff
@@ -22,6 +22,7 @@ public class GameEventHandler {
   private final RoundRepository roundRepo;
   private final GameRepository gameRepo;
   private CardDeck cardDeck;
+  private InstanceCreator instanceCreator;
```

Comment:
```suggestion
  private final ModelFactory modelFactory;
```

Wenn man sich das feld injecten lässt, dann -> Field may be final